foreman.te

```diff
@@ -501,34 +481,183 @@ typealias bin_t alias foreman_hook_t;
+    allow passenger_t websm_port_t:tcp_socket name_connect;
+    allow passenger_t foreman_proxy_port_t:tcp_socket name_connect;
+    corenet_tcp_connect_postgresql_port(passenger_t)
+    corenet_tcp_connect_redis_port(passenger_t)
+    logging_search_logs(passenger_t)
+    logging_list_logs(passenger_t)
+    write_files_pattern(passenger_t, foreman_log_t, foreman_log_t)
+    allow passenger_t foreman_log_t:dir { create_file_perms append_file_perms list_dir_perms search_dir_perms };
+    allow passenger_t foreman_log_t:file { create_file_perms append_file_perms };
+    corenet_tcp_connect_all_ports(passenger_t)
+    files_read_usr_files(passenger_t)
+    term_search_ptys(passenger_t)
+    files_exec_usr_files(passenger_t)
+    dev_read_sysfs(passenger_t)
+    dev_search_sysfs(passenger_t)
+    dev_read_rand(passenger_t)
+    allow passenger_t self:process setsockcreate;
+    miscfiles_manage_cert_dirs(passenger_t)
+    miscfiles_manage_generic_cert_files(passenger_t)
+    can_exec(passenger_t, passenger_tmp_t)
+    admin_pattern(httpd_t, passenger_tmp_t, passenger_tmp_t)
+    admin_pattern(passenger_t, httpd_tmp_t, httpd_tmp_t)
+    corenet_tcp_connect_smtp_port(passenger_t)
+')
+optional_policy(`
+    tunable_policy(`passenger_run_foreman', `
+        postgresql_stream_connect(passenger_t)
+    ')
+')
+optional_policy(`
+    tunable_policy(`passenger_run_foreman', `
+        admin_pattern(httpd_t, foreman_lib_t, foreman_lib_t)
+        admin_pattern(passenger_t, foreman_lib_t, foreman_lib_t)
+        admin_pattern(passenger_t, foreman_var_run_t, foreman_var_run_t)
+        corenet_tcp_connect_virt_port(passenger_t)
+        corenet_tcp_connect_ssh_port(passenger_t)
+        read_files_pattern(passenger_t, foreman_db_t, foreman_db_t)
+        write_files_pattern(passenger_t, foreman_db_t, foreman_db_t)
+        corenet_udp_bind_generic_port(passenger_t)
+        admin_pattern(passenger_t, passenger_tmp_t, passenger_tmp_t)
+    ')
+')
+optional_policy(`
+    tunable_policy(`passenger_run_foreman', `
+        allow passenger_t self:process getsession;
+        fs_rw_anon_inodefs_files(passenger_t)
+        allow passenger_t httpd_t:unix_stream_socket { read write getattr };
+        fs_getattr_xattr_fs(passenger_t)
+        allow passenger_t self:capability2 block_suspend;
+    ')
+')
+optional_policy(`
+    tunable_policy(`passenger_run_foreman', `
+        corenet_tcp_connect_puppet_port(httpd_t)
+        corenet_tcp_connect_puppet_port(passenger_t)
+    ')
+')
+optional_policy(`
+    tunable_policy(`passenger_run_foreman', `
+        corenet_tcp_connect_http_port(httpd_t)
+        corenet_tcp_connect_http_port(passenger_t)
+    ')
+')
+optional_policy(`
+    tunable_policy(`passenger_run_foreman', `
+        mta_send_mail(passenger_t)
+    ')
+')
+optional_policy(`
+    tunable_policy(`passenger_run_foreman', `
+        mysql_stream_connect(passenger_t)
+        mysql_list_db(passenger_t)
+    ')
+')
```

Comment:
MySQL was dropped in 2.0 so no need for this

Reply:
Ok it looks like you did not have too many comments, I will remove everything in the same commit then. Removed.